pinot-core/pom.xml

```diff
@@ -213,5 +213,32 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-math</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
+      <artifactId>parquet-avro</artifactId>
+      <version>${parquet.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
Hadoop brings a lot of dependencies. Its better to avoid pulling in those dependencies  Is there a way to get the schema without depending on Hadoop libs?

If we have to depend on Hadoop then its better to add code into pinot-hadoop module?

Reply:
Ok. It's I'll move it to pinot-hadoop module.

Reply:
> Hadoop brings a lot of dependencies. Its better to avoid pulling in those dependencies Is there a way to get the schema without depending on Hadoop libs?
> 
> If we have to depend on Hadoop then its better to add code into pinot-hadoop module?

I'v refactored code, put parquet related codes in `ParquetUtils.java`, which locates in `pinot-common`, since it already has `hadoop-client` dependency. 
I didn't move it to `pinot-hadoop` because there'll be cross reference between `pinot-core` and `pinot-hadoop`.

Now, `pinot-core` needs `hadoop-client` only in test scope. Is that ok?

Reply:
It's probably ok for now. I cant think of any other work around for now.
 
the hadoop-client dependency in common was added because of hdfs support I think. We plan to remove it once we have the support for deep storage and Uber folks stop using that feature.  Adding more code that depends on this will make it harder for us to remove the dependency on hadoop client.